ethcore/res/ethereum/classic.json

```diff
@@ -25,7 +25,7 @@
 		"accountStartNonce": "0x00",
 		"maximumExtraDataSize": "0x20",
 		"minGasLimit": "0x1388",
-		"networkID" : "0x1",
+		"networkID" : "0x3d",
```

Comment:
Oh, I see. I misinterpreted this as a chainID. Thank you.
Where is CHAIN_ID option btw? I can't find it